lib/pacemaker/pcmk_sched_instances.c

```diff
@@ -778,6 +778,77 @@ unassign_if_mandatory(pe_action_t *first, pe_action_t *then,
     return false;
 }
 
+/*!
+ * \internal
+ * \brief Find first matching action for a clone instance or bundle container
+ *
+ * \param[in] action       Action in an interleaved ordering
+ * \param[in] instance     Clone instance or bundle container being interleaved
+ * \param[in] action_name  Action to look for
+ * \param[in] node         If not NULL, require action to be on this node
+ * \param[in] for_first    If true, \p instance is the 'first' resource in the
+ *                         ordering, otherwise it is the 'then' resource
+ *
+ * \return First action for \p instance (or its containerized resource if a
```

Comment:
Not important since this is explained in detail in the long-form comment in the function body, but "or its containerized resource if a bundle container" reads like the action is always for the containerized resource if `instance` is a bundle container. Of course it's conditional and may be the container itself instead.

Reply:
Worth clarifying